### PR TITLE
BQ tests

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: CT-1315
KBC:  https://github.com/keboola/connection/pull/4905
Driver: https://github.com/keboola/php-storage-driver-bigquery/pull/121
Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

Pokracovanie casti po upravach pre snowflake, teraz pre BQ

### CODE REVIEW

**Comment by romanbracinik** [comment]
tag presiel tu https://dev.azure.com/keboola-dev/Connection%20build/_build/results?buildId=43810&view=logs&j=2e32e92a-b774-5b2a-1ae4-fd294511f88b&t=43215129-de12-593b-bf06-9d12e014da10 spadla tam sox suita ale na inych veciach `Could not read from smtp.sendgrid.net` a podobne

**Review by tomasfejfar — APPROVED** [review]

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php [hunk 1]
```diff
@@ -237,6 +237,124 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
...
+            $schemaName,
+            $schemaName,
+        ));
+
+        /** @var array $tableDataPreview */
+        $tableDataPreview = $this->_client->getTableDataPreview($table['id'], ['format' => 'json']);
+        /** @var array $viewDataPreview1 */
+        $viewDataPreview1 = $this->_client->getTableDataPreview($view['id'], ['format' => 'json']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION'], $tableDataPreview['columns']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION'], $viewDataPreview1['columns']);
```
Comment (romanbracinik) [inline tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:303]: sorry myslel som ze som to tam poslal ale este som robil lepsiu validaciu tak ako to je u snowflaku

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php [hunk 2]
```diff
@@ -237,6 +237,124 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
...
+
+        $this->_client->refreshBucket($idOfBucket);
+
+        // test after refresh, still works
+        /** @var array $tableDataPreview */
+        $tableDataPreview = $this->_client->getTableDataPreview($table['id'], ['format' => 'json']);
+        /** @var array $viewDataPreview1 */
+        $viewDataPreview1 = $this->_client->getTableDataPreview($view['id'], ['format' => 'json']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION', 'AGE'], $tableDataPreview['columns']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION', 'AGE'], $viewDataPreview1['columns']);
```
Comment (romanbracinik) [inline tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:313]: sorry myslel som ze som to tam poslal ale este som robil lepsiu validaciu tak ako to je u snowflaku

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php [hunk 3]
```diff
@@ -237,6 +237,124 @@ public function testRefreshBucketWhenSchemaDoesNotExist(): void
...
+
+        // after refresh should work again
+        $this->_client->refreshBucket($idOfBucket);
+
+        /** @var array $tableDataPreview */
+        $tableDataPreview = $this->_client->getTableDataPreview($table['id'], ['format' => 'json']);
+        /** @var array $viewDataPreview1 */
+        $viewDataPreview1 = $this->_client->getTableDataPreview($view['id'], ['format' => 'json']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION'], $tableDataPreview['columns']);
+        $this->assertSame(['AMOUNT', 'DESCRIPTION'], $viewDataPreview1['columns']);
```
Comment (romanbracinik) [inline tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:355]: sorry myslel som ze som to tam poslal ale este som robil lepsiu validaciu tak ako to je u snowflaku

**Comment by romanbracinik** [comment]
@tomasfejfar davam to este jednou na review sry myslel som si ze som to pushol, su to tie tri veci co som okomentoval dik

**Review by tomasfejfar — APPROVED** [review]

**Pull request merged**